package.json

```diff
@@ -26,8 +26,10 @@
     "node": ">=4.0.0"
   },
   "dependencies": {
+    "big-json": "^2.0.2",
```

Comment:
This library actually uses https://github.com/Faleij/json-stream-stringify under the hood for the actual stringify, so we may be better of using it directly.  However, that library isn't that popular either.

I can't believe this isn't a solved problem in Node in general, but since we keep running into it throughout our tools, it might be time to standard on one library. I found https://www.npmjs.com/package/bfj which has a bunch of people using it and seems well written and thought out and may make for a good general replacement in all of our stuff when it comes to writing large JSON objects.  

We currently have custom JSON writers for large tileset.json in our tilers and elsewhere, so this could replace that. So we could also replace that custom code with bfj.

Thoughts?